pact/coin-contract/v2/coin.pact

```diff
@@ -57,6 +61,11 @@
     "Capability for managing crediting operations"
     (enforce (!= receiver "") "valid receiver"))
 
+  (defcap ROTATE (guard:guard)
```

Comment:
I don't think this will work in signatures, change this to just `account:string`